src/app/appmarket/appinstance/appinstance/appinstance.component.ts

```diff
@@ -116,6 +116,7 @@ export class AppInstanceComponent implements OnInit, OnDestroy {
     public appConfiguration: AppConfiguration;
 
```

Comment:
I think those "checker" variables could be renamed as following to be more intuitive:
- intervalCheckerState
- intervalCheckerHistory